src/services/api/v4/credential-type.service.ts

```diff
@@ -1,5 +1,5 @@
 import { IApiPagedResult } from "@/interfaces/api/result.interface";
-import { ICredentialType } from "@/interfaces/api/v2/credential-type.interface";
+import { ICredentialTypeClaimLabels } from "@/interfaces/api/v4/credential.interface";
```

Comment:
Minor, but the naming implies that the type itself is a label rather than it is a credential type with labels. Wondering if we need to differentiate two types of credential types. Could we instead make the claim labels optional in the base type?

Reply:
consolidated ICredentialType and ICredentialTypeClaimLabels